python/pyspark/pandas/tests/test_numpy_compat.py

```diff
@@ -26,21 +26,17 @@
 
 class NumPyCompatTestsMixin:
     blacklist = [
-        # Koalas does not currently support
+        # Pandas-on-Spark does not currently support
         "conj",
         "conjugate",
         "isnat",
         "matmul",
         "frexp",
         # Values are close enough but tests failed.
-        "arccos",
-        "exp",
-        "expm1",
         "log",  # flaky
         "log10",  # flaky
         "log1p",  # flaky
         "modf",
-        "floor_divide",  # flaky
```

Comment:
For flaky tests, I manually ran whole tests 50 times on local.

`floor_divide` has never failed, so I enabled it.

Other flaky functions failed at least once, so I did not enable them.